testsuite/tests/glbc/test_ingress_reconciliation.py

```diff
@@ -29,18 +27,15 @@ def backend_ingress(request, backend, blame):
 @backoff.on_exception(backoff.fibo, exception=httpx.ConnectError, max_time=600)
 def test_ingress_host_add(backend_ingress):
     """Creates ingress for a backend and checks for host field filled by glbc, checks host points to backend"""
-    rules = backend_ingress.rules
-    assert len(rules) == 1
-
-    backend_ingress.wait_for_hosts()
-    host = rules[0].get("host")
+    backend_ingress.wait_for_certificate()
+    host = backend_ingress.host
 
     assert host
 
-    # needed because of negative dns caching
-    # once https://github.com/kcp-dev/kcp-glbc/issues/354 is implemented this can be removed and reimplemented
-    time.sleep(20)  # wait until dns record is propagated
+    # # needed because of negative dns caching
+    # # once https://github.com/kcp-dev/kcp-glbc/issues/354 is implemented this can be removed and reimplemented
+    # time.sleep(20)  # wait until dns record is propagated
 
-    response = httpx.get(f"http://{host}")
+    response = httpx.get(f"http://{host}", follow_redirects=True)
```

Comment:
why are `follow_redirects=True` needed here? what component does require this?

Reply:
It goes through load balancer and returns 302

Reply:
hmm makes sense but it's weird how it could have worked in the past.